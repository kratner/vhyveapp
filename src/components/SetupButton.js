import React from "react";
import { GoGear } from "react-icons/go";
import styled from "styled-components";

const SetupButtonContainer = styled.div`
  color: #fff;
  cursor: pointer;
  z-index: 10;
  &:hover {
    color: #fdde6b;
  }
`;

const SetupButton = props => {
  return (
    <SetupButtonContainer className="setupbutton">
      <GoGear onClick={props.onClick} size={props.size} title="Setup Options" />
    </SetupButtonContainer>
  );
};

export default SetupButton;
