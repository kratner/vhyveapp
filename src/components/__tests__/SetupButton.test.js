import React from "react";
import { shallow } from "enzyme";
import SetupButton from "../SetupButton";

describe("SetupButton", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<SetupButton />);
    expect(wrapper).toMatchSnapshot();
  });
});
