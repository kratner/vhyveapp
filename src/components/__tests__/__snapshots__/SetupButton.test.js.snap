// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SetupButton renders correctly 1`] = `ShallowWrapper {}`;
